UnitsNet/CustomCode/Wrappers/ReferencePressure.cs

```diff
@@ -25,6 +25,21 @@ namespace UnitsNet.CustomCode.Wrappers
     /// </summary>
     public struct ReferencePressure
     {
+        /// <summary>
+        ///     Represents the pressure at which _pressure is referenced (1 atm default)
```

Comment:
It feels awkward that only pressure is a private field and the other info are public getter properties. I think we can promote `_pressure` to a public getter property and rename it `Pressure`. What do you think?

Reply:
That is a good point. Keeping this private requires processing through the reference conversion to reference pressure under the original measurement reference. Opening it up as a public getter should simplify access to the `Pressure` when simply retrieving the value under the original measurement reference.